$NetBSD$

(1)
patches/patch-ab was there for following fix:
revision 1.9
date: 2008/07/26 06:25:20;  author: obache;  state: Exp;  lines: +14 -2
Start to try support using NextStep (Cocoa or GNUstep) windwing system.
(GNUstep part is not tested yet).

(2) 
patches/patch-ab has been modifying autogen/configure. 
Because of configuration scheme changed a little bit (upstream),
autogen/configure is to be generated from this configure.in.
So the source of autogen/configure or successor of patch-ab
shown above.

--- work/emacs/configure.in	2010-06-08 01:13:11.000000000 +0900
+++ configure.in	2010-06-08 01:13:46.000000000 +0900
@@ -1557,9 +1557,9 @@
   window_system=nextstep
   with_xft=no
   # set up packaging dirs
+  if test "${EN_NS_SELF_CONTAINED}" = yes; then
   exec_prefix=${ns_appbindir}
   libexecdir=${ns_appbindir}/libexec
-  if test "${EN_NS_SELF_CONTAINED}" = yes; then
      prefix=${ns_appresdir}
   fi
   ns_frag=$srcdir/src/ns.mk
